### ISSUE TITLE
ECC support

### ISSUE THREAD

**Issue by nrdmtt** [dalao3/huananzhi-x99-ad3-bios#1]
I have successfully flash the provided bios (thanks a lot for the mod BTW), but the ECC seems now disable. Would it be possible to enable the ECC functionality again? 

PS: I have a V2 mobo, and in my case there is no need to desolder the SPI flash. You just need the power supply to be plugged, in order to give just the 5V to the mobo (ofc, the PC is power off). After that, you can normally use the test clip+ch341 method.

Thanks a lot for your work!

**Comment by dalao3** [comment]
Yes, it does not have ECC support. By reading the following threads, it appears it's hard to get it enabled.
https://www.win-raid.com/t3877f16-X-ECC-support-2.html
https://www.youtube.com/watch?v=HozIT2X-cKk

I would hope someone can port coreboot to these boards and hopefully that will support ECC and fix many bugs.
